### DISCUSSION TITLE
Model Calibration and validation

### DISCUSSION THREAD

**Discussion by fil123** [Universite-Gustave-Eiffel/NoiseModelling#396] [Q&A]
Hi, I would like some suggestions for model validation and calibration. I modeled a study area using traffic flows estimated from annual statistics. At the same time I carried out noise measurements together with the counting of vehicles on various points of the study area. Now I am comparing the measured data (Lm) with those of the model (Lc): I have to make sure that the sum of the squared deviations divided by the number of samples is less than or equal to 0.5.
I have tried to modify TV_D, PVMT (NL01, NL03 ...), absorption of the walls ... but the results are not satisfactory.
Do you have any suggestions, how I could proceed to adapt the calculations to the measured values, so that the deviation is as small as possible.
Thanks.
Philip
At the source:
![image](https://user-images.githubusercontent.com/1425862/142880350-f4b80a9b-136d-4724-b58d-a1935961b826.png)
At receceivers
![image](https://user-images.githubusercontent.com/1425862/142880590-f2ce4b6e-a4a7-4f61-80fc-9d6d10c4c909.png)

![Cattura2](https://user-images.githubusercontent.com/1425862/142881352-b431299f-2ac2-4b74-b48c-24b9243a2890.PNG)

<HTML>
<HEAD>
	
	<META HTTP-EQUIV="CONTENT-TYPE" CONTENT="text/html; charset=utf-8">
	<TITLE></TITLE>
	<META NAME="GENERATOR" CONTENT="OpenOffice 4.1.5  (Win32)">
	<STYLE>
		
	</STYLE>
	
</HEAD>

<BODY TEXT="#000000">
<TABLE FRAME=VOID CELLSPACING=0 COLS=7 RULES=NONE BORDER=0>
	<COLGROUP><COL WIDTH=95><COL WIDTH=95><COL WIDTH=105><COL WIDTH=125><COL WIDTH=134><COL WIDTH=134><COL WIDTH=173></COLGROUP>
	<TBODY>
		<TR>
			<TD WIDTH=95 HEIGHT=17 ALIGN=CENTER>LAEQ_NL0</TD>
			<TD WIDTH=95 ALIGN=CENTER>LAEQ_N03</TD>
			<TD WIDTH=105 ALIGN=CENTER>LAEQ MEAS</TD>
			<TD WIDTH=125 ALIGN=CENTER>VEICH_COUNT</TD>
			<TD WIDTH=134 ALIGN=CENTER>VEICH_MODEL</TD>
			<TD WIDTH=134 ALIGN=LEFT>DELTA LAEQ_NL01</TD>
			<TD WIDTH=173 ALIGN=LEFT>DELTA LAEQ_NL03</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="67,54" SDNUM="1040;">67,54</TD>
			<TD ALIGN=CENTER SDVAL="61,21" SDNUM="1040;">61,21</TD>
			<TD ALIGN=CENTER SDVAL="71,4" SDNUM="1040;">71,4</TD>
			<TD ALIGN=CENTER SDVAL="1528" SDNUM="1040;">1528</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="3,86" SDNUM="1040;">3,86</TD>
			<TD ALIGN=CENTER SDVAL="10,19" SDNUM="1040;">10,19</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="66,04" SDNUM="1040;">66,04</TD>
			<TD ALIGN=CENTER SDVAL="59,71" SDNUM="1040;">59,71</TD>
			<TD ALIGN=CENTER SDVAL="69,6" SDNUM="1040;">69,6</TD>
			<TD ALIGN=CENTER SDVAL="1518" SDNUM="1040;">1518</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="3,55999999999999" SDNUM="1040;">3,56</TD>
			<TD ALIGN=CENTER SDVAL="9,88999999999999" SDNUM="1040;">9,89</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="67,46" SDNUM="1040;">67,46</TD>
			<TD ALIGN=CENTER SDVAL="61,13" SDNUM="1040;">61,13</TD>
			<TD ALIGN=CENTER SDVAL="68" SDNUM="1040;">68</TD>
			<TD ALIGN=CENTER SDVAL="1380" SDNUM="1040;">1380</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="0,540000000000006" SDNUM="1040;">0,54</TD>
			<TD ALIGN=CENTER SDVAL="6,87" SDNUM="1040;">6,87</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="70,19" SDNUM="1040;">70,19</TD>
			<TD ALIGN=CENTER SDVAL="63,77" SDNUM="1040;">63,77</TD>
			<TD ALIGN=CENTER SDVAL="66,7" SDNUM="1040;">66,7</TD>
			<TD ALIGN=CENTER SDVAL="1238" SDNUM="1040;">1238</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="-3,48999999999999" SDNUM="1040;">-3,49</TD>
			<TD ALIGN=CENTER SDVAL="2,93" SDNUM="1040;">2,93</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="69,86" SDNUM="1040;">69,86</TD>
			<TD ALIGN=CENTER SDVAL="63,51" SDNUM="1040;">63,51</TD>
			<TD ALIGN=CENTER SDVAL="66,7" SDNUM="1040;">66,7</TD>
			<TD ALIGN=CENTER SDVAL="1238" SDNUM="1040;">1238</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="-3,16" SDNUM="1040;">-3,16</TD>
			<TD ALIGN=CENTER SDVAL="3,19" SDNUM="1040;">3,19</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="63,13" SDNUM="1040;">63,13</TD>
			<TD ALIGN=CENTER SDVAL="56,61" SDNUM="1040;">56,61</TD>
			<TD ALIGN=CENTER SDVAL="65,6" SDNUM="1040;">65,6</TD>
			<TD ALIGN=CENTER SDVAL="1140" SDNUM="1040;">1140</TD>
			<TD ALIGN=CENTER SDVAL="313" SDNUM="1040;">313</TD>
			<TD ALIGN=CENTER SDVAL="2,46999999999999" SDNUM="1040;">2,47</TD>
			<TD ALIGN=CENTER SDVAL="8,99" SDNUM="1040;">8,99</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="67,04" SDNUM="1040;">67,04</TD>
			<TD ALIGN=CENTER SDVAL="60,71" SDNUM="1040;">60,71</TD>
			<TD ALIGN=CENTER SDVAL="68,6" SDNUM="1040;">68,6</TD>
			<TD ALIGN=CENTER SDVAL="908" SDNUM="1040;">908</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="1,55999999999999" SDNUM="1040;">1,56</TD>
			<TD ALIGN=CENTER SDVAL="7,88999999999999" SDNUM="1040;">7,89</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="69,11" SDNUM="1040;">69,11</TD>
			<TD ALIGN=CENTER SDVAL="62,78" SDNUM="1040;">62,78</TD>
			<TD ALIGN=CENTER SDVAL="63,4" SDNUM="1040;">63,4</TD>
			<TD ALIGN=CENTER SDVAL="824" SDNUM="1040;">824</TD>
			<TD ALIGN=CENTER SDVAL="313" SDNUM="1040;">313</TD>
			<TD ALIGN=CENTER SDVAL="-5,71" SDNUM="1040;">-5,71</TD>
			<TD ALIGN=CENTER SDVAL="0,619999999999997" SDNUM="1040;">0,62</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="70,3" SDNUM="1040;">70,3</TD>
			<TD ALIGN=CENTER SDVAL="63,98" SDNUM="1040;">63,98</TD>
			<TD ALIGN=CENTER SDVAL="67,2" SDNUM="1040;">67,2</TD>
			<TD ALIGN=CENTER SDVAL="798" SDNUM="1040;">798</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="-3,09999999999999" SDNUM="1040;">-3,1</TD>
			<TD ALIGN=CENTER SDVAL="3,22000000000001" SDNUM="1040;">3,22</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="68,44" SDNUM="1040;">68,44</TD>
			<TD ALIGN=CENTER SDVAL="62,12" SDNUM="1040;">62,12</TD>
			<TD ALIGN=CENTER SDVAL="69,9" SDNUM="1040;">69,9</TD>
			<TD ALIGN=CENTER SDVAL="794" SDNUM="1040;">794</TD>
			<TD ALIGN=CENTER SDVAL="313" SDNUM="1040;">313</TD>
			<TD ALIGN=CENTER SDVAL="1,46000000000001" SDNUM="1040;">1,46</TD>
			<TD ALIGN=CENTER SDVAL="7,78000000000001" SDNUM="1040;">7,78</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="67,25" SDNUM="1040;">67,25</TD>
			<TD ALIGN=CENTER SDVAL="60,92" SDNUM="1040;">60,92</TD>
			<TD ALIGN=CENTER SDVAL="68,5" SDNUM="1040;">68,5</TD>
			<TD ALIGN=CENTER SDVAL="752" SDNUM="1040;">752</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="1,25" SDNUM="1040;">1,25</TD>
			<TD ALIGN=CENTER SDVAL="7,58" SDNUM="1040;">7,58</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="66,4" SDNUM="1040;">66,4</TD>
			<TD ALIGN=CENTER SDVAL="59,97" SDNUM="1040;">59,97</TD>
			<TD ALIGN=CENTER SDVAL="70,9" SDNUM="1040;">70,9</TD>
			<TD ALIGN=CENTER SDVAL="752" SDNUM="1040;">752</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="4,5" SDNUM="1040;">4,5</TD>
			<TD ALIGN=CENTER SDVAL="10,93" SDNUM="1040;">10,93</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="69,14" SDNUM="1040;">69,14</TD>
			<TD ALIGN=CENTER SDVAL="62,82" SDNUM="1040;">62,82</TD>
			<TD ALIGN=CENTER SDVAL="69,1" SDNUM="1040;">69,1</TD>
			<TD ALIGN=CENTER SDVAL="736" SDNUM="1040;">736</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="-0,0400000000000063" SDNUM="1040;">-0,04</TD>
			<TD ALIGN=CENTER SDVAL="6,27999999999999" SDNUM="1040;">6,28</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="67,62" SDNUM="1040;">67,62</TD>
			<TD ALIGN=CENTER SDVAL="61,3" SDNUM="1040;">61,3</TD>
			<TD ALIGN=CENTER SDVAL="70,6" SDNUM="1040;">70,6</TD>
			<TD ALIGN=CENTER SDVAL="736" SDNUM="1040;">736</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="2,97999999999999" SDNUM="1040;">2,98</TD>
			<TD ALIGN=CENTER SDVAL="9,3" SDNUM="1040;">9,3</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="66,63" SDNUM="1040;">66,63</TD>
			<TD ALIGN=CENTER SDVAL="60,31" SDNUM="1040;">60,31</TD>
			<TD ALIGN=CENTER SDVAL="64" SDNUM="1040;">64</TD>
			<TD ALIGN=CENTER SDVAL="710" SDNUM="1040;">710</TD>
			<TD ALIGN=CENTER SDVAL="313" SDNUM="1040;">313</TD>
			<TD ALIGN=CENTER SDVAL="-2,63" SDNUM="1040;">-2,63</TD>
			<TD ALIGN=CENTER SDVAL="3,69" SDNUM="1040;">3,69</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="69,19" SDNUM="1040;">69,19</TD>
			<TD ALIGN=CENTER SDVAL="62,87" SDNUM="1040;">62,87</TD>
			<TD ALIGN=CENTER SDVAL="61,7" SDNUM="1040;">61,7</TD>
			<TD ALIGN=CENTER SDVAL="686" SDNUM="1040;">686</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="-7,49" SDNUM="1040;">-7,49</TD>
			<TD ALIGN=CENTER SDVAL="-1,16999999999999" SDNUM="1040;">-1,17</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="65,38" SDNUM="1040;">65,38</TD>
			<TD ALIGN=CENTER SDVAL="59,05" SDNUM="1040;">59,05</TD>
			<TD ALIGN=CENTER SDVAL="67,5" SDNUM="1040;">67,5</TD>
			<TD ALIGN=CENTER SDVAL="682" SDNUM="1040;">682</TD>
			<TD ALIGN=CENTER SDVAL="313" SDNUM="1040;">313</TD>
			<TD ALIGN=CENTER SDVAL="2,12" SDNUM="1040;">2,12</TD>
			<TD ALIGN=CENTER SDVAL="8,45" SDNUM="1040;">8,45</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="70,01" SDNUM="1040;">70,01</TD>
			<TD ALIGN=CENTER SDVAL="63,69" SDNUM="1040;">63,69</TD>
			<TD ALIGN=CENTER SDVAL="70,2" SDNUM="1040;">70,2</TD>
			<TD ALIGN=CENTER SDVAL="664" SDNUM="1040;">664</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="0,189999999999998" SDNUM="1040;">0,19</TD>
			<TD ALIGN=CENTER SDVAL="6,51000000000001" SDNUM="1040;">6,51</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="66,81" SDNUM="1040;">66,81</TD>
			<TD ALIGN=CENTER SDVAL="60,49" SDNUM="1040;">60,49</TD>
			<TD ALIGN=CENTER SDVAL="70,6" SDNUM="1040;">70,6</TD>
			<TD ALIGN=CENTER SDVAL="588" SDNUM="1040;">588</TD>
			<TD ALIGN=CENTER SDVAL="313" SDNUM="1040;">313</TD>
			<TD ALIGN=CENTER SDVAL="3,78999999999999" SDNUM="1040;">3,79</TD>
			<TD ALIGN=CENTER SDVAL="10,11" SDNUM="1040;">10,11</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="68,84" SDNUM="1040;">68,84</TD>
			<TD ALIGN=CENTER SDVAL="62,52" SDNUM="1040;">62,52</TD>
			<TD ALIGN=CENTER SDVAL="67,8" SDNUM="1040;">67,8</TD>
			<TD ALIGN=CENTER SDVAL="564" SDNUM="1040;">564</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="-1,04000000000001" SDNUM="1040;">-1,04</TD>
			<TD ALIGN=CENTER SDVAL="5,27999999999999" SDNUM="1040;">5,28</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="68,98" SDNUM="1040;">68,98</TD>
			<TD ALIGN=CENTER SDVAL="62,66" SDNUM="1040;">62,66</TD>
			<TD ALIGN=CENTER SDVAL="67,8" SDNUM="1040;">67,8</TD>
			<TD ALIGN=CENTER SDVAL="564" SDNUM="1040;">564</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="-1,18000000000001" SDNUM="1040;">-1,18</TD>
			<TD ALIGN=CENTER SDVAL="5,14" SDNUM="1040;">5,14</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="69,97" SDNUM="1040;">69,97</TD>
			<TD ALIGN=CENTER SDVAL="63,64" SDNUM="1040;">63,64</TD>
			<TD ALIGN=CENTER SDVAL="64,5" SDNUM="1040;">64,5</TD>
			<TD ALIGN=CENTER SDVAL="546" SDNUM="1040;">546</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="-5,47" SDNUM="1040;">-5,47</TD>
			<TD ALIGN=CENTER SDVAL="0,859999999999999" SDNUM="1040;">0,86</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="69,61" SDNUM="1040;">69,61</TD>
			<TD ALIGN=CENTER SDVAL="63,28" SDNUM="1040;">63,28</TD>
			<TD ALIGN=CENTER SDVAL="64,5" SDNUM="1040;">64,5</TD>
			<TD ALIGN=CENTER SDVAL="546" SDNUM="1040;">546</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="-5,11" SDNUM="1040;">-5,11</TD>
			<TD ALIGN=CENTER SDVAL="1,22" SDNUM="1040;">1,22</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="67,28" SDNUM="1040;">67,28</TD>
			<TD ALIGN=CENTER SDVAL="60,95" SDNUM="1040;">60,95</TD>
			<TD ALIGN=CENTER SDVAL="66,4" SDNUM="1040;">66,4</TD>
			<TD ALIGN=CENTER SDVAL="456" SDNUM="1040;">456</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="-0,879999999999995" SDNUM="1040;">-0,88</TD>
			<TD ALIGN=CENTER SDVAL="5,45" SDNUM="1040;">5,45</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="72" SDNUM="1040;">72</TD>
			<TD ALIGN=CENTER SDVAL="65,68" SDNUM="1040;">65,68</TD>
			<TD ALIGN=CENTER SDVAL="65" SDNUM="1040;">65</TD>
			<TD ALIGN=CENTER SDVAL="388" SDNUM="1040;">388</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="-7" SDNUM="1040;">-7</TD>
			<TD ALIGN=CENTER SDVAL="-0,680000000000007" SDNUM="1040;">-0,68</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="67,76" SDNUM="1040;">67,76</TD>
			<TD ALIGN=CENTER SDVAL="61,44" SDNUM="1040;">61,44</TD>
			<TD ALIGN=CENTER SDVAL="64,3" SDNUM="1040;">64,3</TD>
			<TD ALIGN=CENTER SDVAL="368" SDNUM="1040;">368</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="-3,46000000000001" SDNUM="1040;">-3,46</TD>
			<TD ALIGN=CENTER SDVAL="2,86" SDNUM="1040;">2,86</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="69,8" SDNUM="1040;">69,8</TD>
			<TD ALIGN=CENTER SDVAL="63,48" SDNUM="1040;">63,48</TD>
			<TD ALIGN=CENTER SDVAL="66" SDNUM="1040;">66</TD>
			<TD ALIGN=CENTER SDVAL="336" SDNUM="1040;">336</TD>
			<TD ALIGN=CENTER SDVAL="313" SDNUM="1040;">313</TD>
			<TD ALIGN=CENTER SDVAL="-3,8" SDNUM="1040;">-3,8</TD>
			<TD ALIGN=CENTER SDVAL="2,52" SDNUM="1040;">2,52</TD>
		</TR>
		<TR>
			<TD HEIGHT=17 ALIGN=CENTER SDVAL="72,51" SDNUM="1040;">72,51</TD>
			<TD ALIGN=CENTER SDVAL="66,19" SDNUM="1040;">66,19</TD>
			<TD ALIGN=CENTER SDVAL="64,1" SDNUM="1040;">64,1</TD>
			<TD ALIGN=CENTER SDVAL="296" SDNUM="1040;">296</TD>
			<TD ALIGN=CENTER SDVAL="625" SDNUM="1040;">625</TD>
			<TD ALIGN=CENTER SDVAL="-8,41000000000001" SDNUM="1040;">-8,41</TD>
			<TD ALIGN=CENTER SDVAL="-2,09" SDNUM="1040;">-2,09</TD>
		</TR>
	</TBODY>
</TABLE>
</BODY>

</HTML>

**Comment by pierromond** [comment]
Hi !

Could you give us more parameters ? 

Do you use Leq or Lden values ?
Which configuration paramters ? reflexion ? diffraction ?
The mean error is quite ok with NL01 (-1.2 dB), maybe the pavement is not the same for the whole study area ?
etc. etc.
